net/ooniprobe/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ooniprobe
+PKG_VERSION:=3.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=probe-cli-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ooni/probe-cli/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=d1e8c376d89bd7aa3a0971eac95abf1bbd438df072f7f9e5b9e32d8753ff3f71
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE.md
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/probe-cli-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=golang/host measurement-kit
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/ooni/probe-cli
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/ooniprobe
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=OONI probe-cli
+  URL:=https://ooni.org
+  DEPENDS:=\
+	$(GO_ARCH_DEPENDS) +libevent2-openssl \
+	+libevent2-core +libevent2-extra +libevent2-pthreads \
+	+libmaxminddb +libnghttp2 +libopenssl \
```

Comment:
Are you sure libnghttp2 is necessary?

Reply:
Without it I get 
```
Package ooniprobe is missing dependencies for the following libraries:
libnghttp2.so.14
```

Reply:
I have compiled this package without `+libnghttp2` in `DEPENDS` (and without the `-lnghttp2` in `TARGET_LDFLAGS`) in armvirt-64 and x86-generic, and I haven't encountered this error... Is there a specific target arch I should try compiling for to see this error?

Reply:
I use mvebu which should be arm32 . I think it could be somehow related to https://github.com/ooni/probe-engine/commit/ae59c0578995e64d7175267db53bf7bc8bccca31#diff-38f5d43a0af2601dc205a09e9e321845

Reply:
When we compile for Linux without the "ooni" flag (which does what we want in our strange setup for Linux) we just `-lcurl` and assume the dynamic linker will do the rest.

When I did that, I assumed a distro would have preferred dynamic linking. Is there a reason not to use dynamic linking of C/C++ dependencies here?

Reply:
Dynamic linking to curl here is fine; `TARGET_LDFLAGS` has `-lcurl`.

I was asking about libnghttp2 - it doesn't appear to be necessary anymore in the latest release.

Reply:
So, MK never depended directly from nghttp2. AFAIK, however, curl uses nghttp2 to implement http2. Hence, whether it's needed or not I think depends on libcurl.so?